scripts/commands.py

```diff
@@ -165,8 +165,9 @@ def google_official_search(query, num_results=8):
     return search_results_links
 
 def browse_website(url, question):
-    summary = get_text_summary(url, question)
-    links = get_hyperlinks(url)
+    website_content = browse.get_website_content(url)
```

Comment:
please remove these changes for commands.py to keep the PR atomic and related to the title of the PR

Reply:
@richbeales my apologies, I have dropped the changes. It came from my previous PR.  I think it should be fine now.